tests/AssertAnnotationTest.php

```diff
@@ -1474,6 +1555,17 @@ function getPropertyType(\ReflectionProperty $reflectionItem): ?string {
                 [],
                 '7.4',
             ],
+            'PHP80-reflectionNameTypeClassStringIfNotBuiltin' => [
+                '<?php
+                    /** @return class-string|"self"|"static"|null */
+                    function getPropertyType(\ReflectionProperty $reflectionItem): ?string {
+                        $type = $reflectionItem->getType();
+                        return ($type instanceof \ReflectionNamedType) && !$type->isBuiltin() ? $type->getName() : null;
+                    }',
+                [],
+                [],
+                '8.0',
+            ],
```

Comment:
Disregard this test at the moment pls. It's from another PR, didn't clean. Will do later